spec/21-http_response_header_format.md

```diff
@@ -103,10 +103,11 @@ The second least significant bit of the trace-flags field denotes the random-tra
 If a trace was started by a downstream participant and it responds with the `traceresponse` HTTP header, an upstream participant can use this flag to determine if the `trace-id` was generated as per
 the specification for this flag.
 
-The format and requirements for this are the same as those of the random-trace-id flag in the trace-flags field in the `traceparent` request header.
+When a participant starts or restarts a trace (that is, when the participant generates a new `trace-id`), the requirements for this flag are the same as those for the random-trace-id flag in the trace-flags field in the `traceparent` request header. For details, see the section [Random Trace ID Flag](#random-trace-id-flag).
 
-For details, see the trace-flags section under [traceparent Header Field Values](#traceparent-Header-Field-Values).
```

Comment:
I removed this link. Instead, this section now links directly to the random-trace-id section in the request header section. This section has all the rules and information that are relevant here. IMO the only reason to link to the whole traceparent-header-field-values section would be if we also wanted to include this line from the trace-id subsection: 

https://github.com/w3c/trace-context/blob/main/spec/20-http_request_header_format.md?plain=1#L108

> If the right-most 7 bytes are randomly (or pseudo-randomly) generated, the corresponding [random trace id flag](#random-trace-id-flag) SHOULD be set.

But that particular rule is [repeated](https://github.com/w3c/trace-context/blob/main/spec/20-http_request_header_format.md?plain=1#L191) in the random-trace-id section as well.